source/scripting_v3/GTA/Entities/Vehicles/Vehicle.cs

```diff
@@ -61,6 +61,16 @@ public new bool Exists()
 		public bool HasDonkHydraulics => Game.Version >= GameVersion.v1_0_505_2_Steam && SHVDN.NativeMemory.HasVehicleFlag(Model.Hash, NativeMemory.VehicleFlag2.HasLowriderDonkHydraulics);
 		public bool HasParachute => Game.Version >= GameVersion.v1_0_505_2_Steam && Function.Call<bool>(Hash._GET_VEHICLE_HAS_PARACHUTE, Handle);
 		public bool HasRocketBoost => Game.Version >= GameVersion.v1_0_944_2_Steam && Function.Call<bool>(Hash._GET_HAS_ROCKET_BOOST, Handle);
+		public bool IsRocketBoostActive
+		{
+			get => Function.Call<bool>((Hash)0x3D34E80EED4AE3BE, Handle);
+			set => Function.Call((Hash)0x81E1552E35DC3839, Handle, value);
+		}
+		public bool IsParachuteActive
+		{
+			get => Function.Call<bool>((Hash)0x3DE51E9C80B116CF, Handle);
+			set => Function.Call((Hash)0x0BFFB028B3DD0A97, Handle, value);
+		}
```

Comment:
Check the game version first before calling natives since these are present only in newer versions. For consistency with extsting properties, if the game runs in older versions, return `false` from the getter method and throw `GameVersionNotSupportedException` from the setter method ([an example](https://github.com/crosire/scripthookvdotnet/blob/e726766f793a299ef710cfed9bf89e2fc9c20bb1/source/scripting_v3/GTA/Entities/Vehicles/VehicleModCollection.cs#L317)). We are supposed to return `false` in older versions when features are supported only in the newer game versions, just like most of natives like `IS_*` usually return `false` if the passed handle is not valid iirc.
For your information, `IS_ROCKET_BOOST_ACTIVE` and `SET_ROCKET_BOOST_ACTIVE` and `VEHICLE_START_PARACHUTING` are present in 944 or later versions, `IS_VEHICLE_PARACHUTE_DEPLOYED` is present in 1011 or later versions.
I believe there is a way to check if the parachute of vehicle is deployed without `IS_VEHICLE_PARACHUTE_DEPLOYED` so we can check the parachute state in b944, but that's another story and I'll do it if needed.